src/basic/random-util.c

```diff
@@ -51,6 +53,7 @@ static void fallback_random_bytes(void *p, size_t n) {
 
         memcpy(state.label, "systemd fallback random bytes v1", sizeof(state.label));
```

Comment:
Not a matter, but version should be increased?

Reply:
i guess that would make sense if we would not also increase the size of the structure. but we do, so i doubt that#s necessary.